c10/util/order_preserving_flat_hash_map.h

```diff
@@ -741,6 +763,7 @@ class sherwood_v3_table : private EntryAlloc, private Hasher, private Equal
             if (it->has_value())
                 it->destroy_value();
         }
+        reset_list();
```

Comment:
What is `sentinel` when the list is empty if `reset_list` makes it point to itself?